Docs/source/usage/parameters.rst

```diff
@@ -2966,11 +2966,14 @@ In-situ capabilities can be used by turning on Sensei or Ascent (provided they a
 * ``<diag_name>.<species_name>.variables`` (list of `strings` separated by spaces, optional)
     List of particle quantities to write to output.
     Choices are ``x``, ``y``, ``z`` for the particle positions (3D and RZ), ``x`` & ``z`` in 2D, ``z`` in 1D,
-    ``w`` for the particle weight and ``ux``, ``uy``, ``uz`` for the particle momenta.
+    ``w`` for the particle weight, ``ux``, ``uy``, ``uz`` for the particle momenta, and ``Ex``, ``Ey``, ``Ez``, ``Bx``, ``By``, ``Bz`` for the electromagnetic fields.
     When using the lab-frame electrostatic solver, ``phi`` (electrostatic potential, on the macroparticles) is also available.
-    By default, all particle quantities (except ``phi``) are written.
+    By default, positions, momenta and particle weight are written.
     If ``<diag_name>.<species_name>.variables = none``, no particle data are written.
 
+    .. note::
+        The electromagnetic fields diagnostics will not display the external fields.
```

Comment:
Maybe you could write something like "will not include" instead of "will not display"